### ISSUE TITLE
Not connected 

### ISSUE THREAD

**Issue by acabamar** [BTCPrivate/electrum-btcp#97]
I spent two days with the electrum wallet P! 1.1.1 in the "not connected" state.  How do I solve that and switch to connected status?

**Comment by stevewig79** [comment]
Try this (remove the tick)

[cid:image001.png@01D516DA.01998EA0]

**Comment by calql8tor** [comment]
I am having the same issue.  What tick are you referring to?

**Comment by stevewig79** [comment]
The ‘Select server automatically’ tick. Although, mine won’t sync now either :-(

-sтεvε ωιggιηs-




On 31 May 2019, at 16:36, calql8tor <notifications@github.com<mailto:notifications@github.com>> wrote:


I am having the same issue. What tick are you referring to?

**Comment by Aurelchan** [comment]
Same issue here. No sync anymore.

**Comment by wmsimru** [comment]
An updated electrum server was launched thanks to our wonderful supportive community. If you are experiencing issues please use the hostname: http://electrum3.btcprivate.org    port: 5222 

TOOLS -> NETWORK -> SERVER

If my comment was helpful - I will be grateful for a gift:
BTCP - b1KZv8YvfUQSR5yiVXeZgg11oa8kmok4c8Y

**Comment by timothy7285** [comment]
Sever http://electrum3.btcprivate.org port: 5222 not working anymore, anyone if there's working severs?

**Comment by interbiznw** [comment]
@timothy7285 why not use, electrum.btcprivate.org port: 5222 ??

**Comment by timothy7285** [comment]
@interbiznw both electrum3.btcprivate.org  port: 5222 and electrum.btcprivate.org port: 5222 not connecting for me. Any suggestions on what the issue could be?

**Comment by interbiznw** [comment]
@timothy7285 when was the last time you know it worked? Could be outdated certs. Are you on windows?

**Comment by timothy7285** [comment]
@interbiznw last used it a few months ago. I'm on windows 10.

**Comment by interbiznw** [comment]
1) Close electrum

2) Delete the certs located in
`%appdata%/electrum-btcp/certs/`

3) restart electrum, should connect to electrum.btcprivate.org   port: 5222

**Comment by timothy7285** [comment]
@interbiznw thank you seems to connect now, it's the first time I've had a problem with the cert being outdated

**Comment by interbiznw** [comment]
@timothy7285 Yeah won't be a regular thing, its just we changed servers, and the cert changed. The old cert being in your folder did not match so it would not accept the connection. Clearing them downloads the current cert. Glad that worked!

**Comment by pankajpatil12** [comment]
Thanks, this also works on Mac by removing files from ~/.electrum-btcp/certs/ and restart electrum.

**Comment by TheFedex87** [comment]
I have same problem, I have a Electrum BTCP wallet create on 02-2018. Now I tried to open it again with the latest version of wallet, but it won't connect! Red status! Help please :(

**Comment by interbiznw** [comment]
> I have same problem, I have a Electrum BTCP wallet create on 02-2018. Now I tried to open it again with the latest version of wallet, but it won't connect! Red status! Help please :(

https://twitter.com/bitcoinprivate/status/1223300679642624001

**Comment by TheFedex87** [comment]
> > I have same problem, I have a Electrum BTCP wallet create on 02-2018. Now I tried to open it again with the latest version of wallet, but it won't connect! Red status! Help please :(
> 
> https://twitter.com/bitcoinprivate/status/1223300679642624001

Thank you very much for reply. Honestly I didn't understand very well. What does that mean? I had some btcp in an electrum btcp wallet. Have I lost them? How can I move them?

**Comment by interbiznw** [comment]
No they are not lost, you have the ability to access your private keys whether the wallet can connect or not, the keys are in your wallet and can be accessed offline. The coins are on the blockchain and were snapshotted at block 680000 and will be issued on the new chain. There is no release date for that yet just stay tuned to our social media channels.. twitter, telegram medium or discord and we will give updates on the new chain release as soon as we know more.

> > > I have same problem, I have a Electrum BTCP wallet create on 02-2018. Now I tried to open it again with the latest version of wallet, but it won't connect! Red status! Help please :(
> > 
> > 
> > https://twitter.com/bitcoinprivate/status/1223300679642624001
> 
> Thank you very much for reply. Honestly I didn't understand very well. What does that mean? I had some btcp in an electrum btcp wallet. Have I lost them? How can I move them?

**Comment by jonasman** [comment]
Any news here? i cannot connect

**Comment by interbiznw** [comment]
> Any news here? i cannot connect

Same news as before..

https://twitter.com/bitcoinprivate/status/1223300679642624001

**Comment by Galag-en** [comment]
Hello people, How do I withdraw my BTCP I can't connect to the wallet, is there a way? https://prnt.sc/11bh9le

**Comment by SnanCakmak** [comment]
Hello there
My account is not connected since 2018, I could not solve this situation in any way, I cleared the certifica, I entered the given server address again.
